packages/docs/src/components/header/components/HeaderSelector.vue

```diff
@@ -44,12 +44,12 @@ export default defineComponent({
   },
 
   setup (props, { emit }) {
-    const { getColors } = useColors()
+    const { colors } = useColors()
 
     const toggleSidebar = () => emit('toggleSidebar', !props.minimized)
 
     return {
-      colors: computed(getColors),
+      colors,
```

Comment:
@aluarius, thank you very much for your help. I take a new way to solve issue 2266. There are some places to change in similar way, So I have pick one place as an example for discussion. I have changed `colors: computed(getColors)` to `colors`, is this the intended solution?